codefellowship/codefellowship/src/main/resources/templates/userdetails.html

```diff
@@ -16,12 +16,13 @@ <h2 id="users pronouns" th:text="${user.pronouns}"></h2>
         <div id="users operating system" th:text="${user.os}"></div>
         <div id="user biography" th:text="${user.bio}"></div>
     </section>
-    <!--    <section id="user posts">-->
-    <!--        <div th:each="post : ${user.posts}"></div>-->
-    <!--        <h3 th:text="${post.title}"></h3>-->
-    <!--        <div th:text="${post.body}"></div>-->
-    <!--        <div th:text="${post.timestamp}"></div>-->
-    <!--    </section>-->
+
+    <section id="user posts">
```

Comment:
This will crash if the list of posts is empty. Add a th:if="${user.posts.size() > 0}" to the section tag.